openquakeplatform/openquakeplatform/test/platform.py

```diff
@@ -0,0 +1,233 @@
+import time
+from utils import *
+
+
+class Platform(object):
+    DT = 0.1
+
+    def __init__(self):
+
+        self.driver = None
+        self.basepath = None
+        self.user = None
+        self.passwd = None
+        self.debugger = None
+
+    def init(self):
+        try:
+            from openquakeplatform.test.config import (
+                pla_basepath, pla_user, pla_passwd, pla_debugger)
+        except ImportError:
+            print (
+                "ERROR: config.py not found. Copy config.py.tmpl in config.py "
+                " and modify it properly.")
+            sys.exit(1)
+
+        self.debugger = pla_debugger
+        self.driver = self.driver_create("firefox", self.debugger)
+        self.basepath = pla_basepath
+        self.user = pla_user
+        self.passwd = pla_passwd
+
+        self.driver.maximize_window()
+        self.homepage_login()
+
+    @staticmethod
+    def driver_create(name, debugger):
+        if name == "firefox":
+            fp = webdriver.FirefoxProfile()
+            if debugger != "":
+                fp.add_extension(extension=debugger)
+                fp.native_events_enabled = False
+                fp.set_preference("browser.tabs.warnOnClose", False)
+                fp.set_preference("extensions.firebug.allPagesActivation",
+                                  "on")
+                fp.set_preference("extensions.firebug.console.enableSites",
+                                  True)
+                fp.set_preference("extensions.firebug.defaultPanelName",
+                                  "console")
+            driver = webdriver.Firefox(firefox_profile=fp)
+        else:
+            driver = None
+
+        return driver
+
+    def homepage_login(self):
+        # "" is the (empty) path for the page required for login
+        self.driver.get(self.basepath + "")
```

Comment:
I really do not understand this. Please remove the empty string and add a comment to make the intent clear.